apps/site/src/components/layout/Header.astro

```diff
@@ -0,0 +1,103 @@
+---
+import { Icon } from "astro-icon/components";
+import Logo from "./Logo.astro";
+
+const links = [
+  {
+    href: "/",
+    text: "Home",
+  },
+  {
+    href: "/projects",
+    text: "Projects",
+  },
+  {
+    href: "/docs",
+    text: "Documentation",
+  },
+  {
+    href: "/blog",
+    text: "Articles",
+  },
+];
+
+const socialLinks = [
+  {
+    href: "/",
+    text: "Twitter (X)",
+    icon: "ri:twitter-x-fill",
+  },
+  {
+    href: "/docs",
```

Comment:
```suggestion
    href: "https://github.com/astrolicious",
```

Reply:
I disagree with that. I think since the starlight docs are really wip for now, we can just comment it